### DISCUSSION TITLE
大佬门，这是什么问题，qaq

### DISCUSSION THREAD

**Discussion by pengpaiz** [BewlyBewly/BewlyBewly#1106] [General]
![屏幕截图 2024-11-02 230822](https://github.com/user-attachments/assets/04d8e61b-d1cf-42d2-afb0-9751423d4978)

**Comment by hakadao** [comment]
提供一下bewly版本號

**Reply by pengpaiz** [reply]
[v0.34.2](https://github.com/BewlyBewly/BewlyBewly/releases/tag/v0.34.2)我试过好多版本，都遇到这个问题

**Comment by hakadao** [comment]
edge也可以用chrome web store下載，本地下載時不要選帶-firefox後綴的如果你是edge

**Reply by pengpaiz** [reply]
已解决，感谢

**Reply by pengpaiz** [reply]
![屏幕截图 2024-11-06 212947](https://github.com/user-attachments/assets/1dd3d7b8-3331-4108-9871-ff92c10aa98e)
大佬，我又碰到问题了，主页为什么一些视频的封面未显示，网络正常
